### PR TITLE
added chrome web push notification support

### PR DESCRIPTION
**Pull request by pankaj-kumar34**

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -1074,6 +1074,16 @@
         "verified_result": null
       }
     ],
+    "ibm/resource_ibm_push_notification_chrome.go": [
+      {
+        "hashed_secret": "8a4036fbf3e13c7a84bac77ddae25de1cdfc4d43",
+        "is_secret": false,
```
Comment (pankaj-kumar34) [inline .secrets.baseline:1080]: This is not sensitive data.

File: ibm/resource_ibm_push_notification_chrome.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	}
+
+	guid := d.Id()
+
+	chromeWebConf, response, err := pnClient.GetChromeWebConf(&pushservicev1.GetChromeWebConfOptions{
+		ApplicationID: &guid,
+	})
+
+	if err != nil {
+		if response.StatusCode == 404 {
```
Comment (hkantare) [inline ibm/resource_ibm_push_notification_chrome.go:85]: add condition check response != nil && response.StatusCode == 404

File: ibm/resource_ibm_push_notification_chrome.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+		return err
+	}
+	guid := d.Get("guid").(string)
+
+	response, err := pnClient.DeleteChromeWebConf(&pushservicev1.DeleteChromeWebConfOptions{
+		ApplicationID: &guid,
+	})
+
+	if err != nil {
+		if response.StatusCode == 404 {
```
Comment (hkantare) [inline ibm/resource_ibm_push_notification_chrome.go:113]: response != nil && response.StatusCode == 404

File: website/docs/d/pn_application_chrome.html.markdown [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+layout: "ibm"
+page_title: "IBM : pn_application_chrome"
+subcategory: "Push Notifications"
```
Comment (hkantare) [inline website/docs/d/pn_application_chrome.html.markdown:4]: subcategory: "Push Notifications" should be first stmt before layout

File: website/docs/r/pn_application_chrome.html.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+layout: "ibm"
+page_title: "IBM : pn_application_chrome"
+subcategory: "Push Notifications"
```
Comment (hkantare) [inline website/docs/r/pn_application_chrome.html.markdown:4]: subcategory: "Push Notifications" should be first stmt before layout

**Pull request merged**